tests/neo-asm.Tests/Linker/Linker.cs

```diff
@@ -0,0 +1,89 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace neo_asm.Tests.Linker
+{
+    [TestClass]
+    public class Linker
+    {
+        [TestMethod]
+        public void TestBaseCode()
+        {
+            const string testcode = @"
+/*comment1*///comment 2
+Main()
+{
+  PUSH 1
+  PUSH 2
+  ADD
+  RET
+}
+";
+
+            var code = Neo.ASML.Parser.WordScanner.CreateSourceCode("a.asm", testcode);
+            var proj = Neo.ASML.Parser.Parser.Parse(code);
+            var module = Neo.ASML.Linker.Linker.CreateModule(proj);
+            var avm = Neo.ASML.Linker.Linker.Link(module, "Main");
+
+            module.Dump((str) => Console.WriteLine(str));
+            Console.WriteLine("avm=" + Helper.Hex2Str(avm));
+            //check output
```

Comment:
This test seems a little 'weak', would it be too hard to check the resulting avm to see if it is correct?